cli/src/commands/run.rs

```diff
@@ -39,4 +41,8 @@ pub struct RunArgs {
     /// This will trigger processing(=pie generation) step
     #[arg(short, long, requires("program_input_file"))]
     pub cairo_pie_file: Option<PathBuf>,
+
+    /// Destination chain id
+    #[arg(long)]
+    pub destination_chain_id: Option<ChainId>,
```

Comment:
For `hdp run` command, as request json format ([`SubmitBatchQuery`](https://github.com/HerodotusDev/hdp/blob/e9a5b67ece719a9192b3c071b770ca86515d3cf0/hdp/src/primitives/request.rs#L15) struct) already contains destination chain id as necessary information. So imo we don't need to pass as argument instead parse from file only.

Reply:
Makes sense. Will remove

Reply:
*Removed*